lib/nodejs/vwf.js

```diff
@@ -7,7 +7,9 @@ var parseurl = require( './parse-url' ),
     servehandler = require( './serve-handler' ),
     helpers = require( './helpers' ),
     application = require( './application' ),
-    url = require( 'url' );
+    url = require( 'url' ),
+    lobby = require( './lobby' ),
+    finalhandler = require( 'express/node_modules/finalhandler' );
```

Comment:
Is it common practice to reach into the `node_modules` of `express` to specify a dependency rather than adding it as our own dependency in `package.json`?